### PR TITLE
Fix formatting issues in NGINX, Redis and MySQL templates

### PR DESCRIPTION
**Pull request by acearun**
1. Fix column names
2. Update chart formatting where applicable.
3. Add Uptime column to MySQL overview.

### CODE REVIEW

File: Workbooks/Workloads/Redis Insights/Redis Insights.workbook [hunk 1]
```diff
@@ -548,6 +549,30 @@
...
+                "sortBy": [
+                  {
+                    "itemKey": "UsedRatio",
+                    "sortOrder": 2
+                  }
+                ],
+                "labelSettings": [
+                  {
+                    "columnId": "Computer",
+                    "label": "Virtual machines"
```
Comment (dougbrad) [inline Workbooks/Workloads/Redis Insights/Redis Insights.workbook:562]: Wondering - when we support Arc - we change back to Computer ?

Comment (acearun) [reply Workbooks/Workloads/Redis Insights/Redis Insights.workbook:562]: I think we should go with more generic label to include VMs, Containers and Arc machines.

File: Workbooks/Workloads/NGINX Insights/NGINX Insights.workbook [hunk 2]
```diff
@@ -411,6 +411,36 @@
...
+                    "columnId": "WaitingRequestsRate",
+                    "label": "Waiting request rate"
+                  },
+                  {
+                    "columnId": "WaitingTrend",
+                    "label": "Waiting request trend"
+                  },
+                  {
+                    "columnId": "Cpu",
+                    "label": "Cpu utilization"
```
Comment (dougbrad) [inline Workbooks/Workloads/NGINX Insights/NGINX Insights.workbook:438]: I think standard is 'CPU' ?

Comment (acearun) [reply Workbooks/Workloads/NGINX Insights/NGINX Insights.workbook]: Reverted back to Cpu

**Pull request merged**